### DISCUSSION TITLE
I wrote a "package manager" for FreshRSS extensions

### DISCUSSION THREAD

**Discussion by yzqzss** [FreshRSS/FreshRSS#6210] [Show and tell]
https://github.com/yzqzss/freshrss-ext-manager

It's too cumbersome to manually install/upgrade extensions one by one. So I wrote this simple tool to help me manage extensions :)

**Comment by Aevrin** [comment]
Tested, really nice.  If this could be added to the extensions page (as a webpage), that would be really cool because it would make extension management part of managing the system as a whole, and not a completely separate task.  Are you releasing this as open source?

**Reply by yzqzss** [reply]
It's technically possible, but I do not know PHP, so... :(

> Are you releasing this as open source?

Yes, it's under AGPLv3 now.
